### PR TITLE
DG Update

### PR DESCRIPTION
**Pull request by kashing555**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -541,6 +542,31 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+    * 1a1. TrackBeau shows an error message.
+
+      Use case resumes at step 2.
+
+* 1b. The given parameters is invalid.
+    * 1b1. TrackBeau shows an error message.
+
+      Use case resumes at step 2.
+
+* 1c. The list is empty.
```
Comment (flairekq) [inline docs/DeveloperGuide.md:564]: Oh hmm should this be 2a instead? Cause list is shown at step 2 :O

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -664,7 +714,7 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 * *a. At any time, User requests to [view help UC01](#use-case-uc01---view-help)
 
 
-#### Use case: UC11 - List bookings
+#### Use case: UC12 - List bookings
```
Comment (flairekq) [inline docs/DeveloperGuide.md:717]: Can update the link to this use case? cause previously was [list bookings UC11](#use-case-uc11---list-bookings) now need to change to [list bookings UC12](#use-case-uc12---list-bookings)

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -449,37 +468,20 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
-**Extensions:**
-
-* 2a. The list is empty.
-
-  Use case ends.
-
-* *a. At any time, User requests to [view help UC01](#use-case-uc01---view-help)
-
-#### Use case: UC03 - List customers
+#### Use case: UC02 - List customers
```
Comment (flairekq) [inline docs/DeveloperGuide.md:471]: Can update the link to this use case? Previously was [list customers UC03](#use-case-uc03---list-customers) now becomes [list customers UC02](#use-case-uc02---list-customers)

**Comment by hanhuiice** [comment]
LGTM!

**Review by flairekq — CHANGES_REQUESTED** [review]
Some changes required

**Pull request merged**